tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -571,44 +575,44 @@ tf_cc_test(
     ],
 )
 
-cc_library(
-    name = "cusolver_context",
-    srcs = ["cusolver_context.cc"],
-    hdrs = ["cusolver_context.h"],
-    deps = [
-        "//tensorflow/compiler/xla:statusor",
-        "//tensorflow/compiler/xla:types",
-        "//tensorflow/compiler/xla:util",
-        "//tensorflow/core:lib",
-        "//tensorflow/core:stream_executor_no_cuda",
-        "//tensorflow/stream_executor:blas",
-    ]
-    + if_cuda_is_configured(if_cuda([
-        "@local_config_cuda//cuda:cuda_headers",
-        "@local_config_cuda//cuda:cusolver",
-    ])),
-)
+#cc_library(
+#    name = "cusolver_context",
+#    srcs = ["cusolver_context.cc"],
+#    hdrs = ["cusolver_context.h"],
+#    deps = [
+#        "//tensorflow/compiler/xla:statusor",
+#        "//tensorflow/compiler/xla:types",
+#        "//tensorflow/compiler/xla:util",
+#        "//tensorflow/core:lib",
+#        "//tensorflow/core:stream_executor_no_cuda",
+#        "//tensorflow/stream_executor:blas",
+#    ]
+#    + if_cuda_is_configured(if_cuda([
+#        "@local_config_cuda//cuda:cuda_headers",
+#        "@local_config_cuda//cuda:cusolver",
+#    ])),
+#)
 
-cc_library(
-    name = "cusolver_rewriter",
-    srcs = ["cusolver_rewriter.cc"],
-    hdrs = ["cusolver_rewriter.h"],
-    deps = [
-        ":cusolver_context",
-        ":ir_emission_utils",
-        ":scratch_allocator",
-        "//tensorflow/compiler/xla:literal",
-        "//tensorflow/compiler/xla:util",
-        "//tensorflow/compiler/xla:xla_data_proto",
-        "//tensorflow/compiler/xla/service:device_memory_allocator",
-        "//tensorflow/compiler/xla/service:hlo",
-        "//tensorflow/compiler/xla/service:hlo_pass",
-        "//tensorflow/core:lib",
-        "//tensorflow/core:stream_executor_no_cuda",
-        "//tensorflow/stream_executor:blas",
-        "@com_google_absl//absl/types:optional",
-    ],
-)
+#cc_library(
+#    name = "cusolver_rewriter",
```

Comment:
do we need to comment out this target? perhaps use `if_cuda_is_configured()` to make `srcs` and `hdrs` empty?